app/javascript/components/ResultsPanel.js

```diff
@@ -1,51 +1,25 @@
 import React, { useContext } from 'react'
-import Tab from 'react-bootstrap/lib/Tab'
-import Tabs from 'react-bootstrap/lib/Tabs'
 import { StudySearchContext } from 'components/search/StudySearchProvider'
-
+import StudyResults from 'components/StudyResults'
```

Comment:
Imports like this that rely on rules defined in Rails `webpacker.yml` prevent `Go to Definition` in VS Code (and presumably other IDEs) by default.

They also prevent automatic refactoring, e.g. you couldn't `Rename Symbol` in `StudyResults.js` and have that propagate to `ResultsPanel.js`.

So, for better developer ergonomics, I'd suggest:

```suggestion
import StudyResults from './StudyResults'
```